source/mainnet/smart-contracts/onboarding-guide-ethereum-developers/faq.rst

```diff
@@ -0,0 +1,371 @@
+.. _faq:
+
+====
+FAQs
+====
+
+This is a list of Frequently Asked Questions about Concordium. It is focused on
+helping developers with an Ethereum/solidity background to understand
+the Concordium blockchain and its smart contract ecosystem.
+
+Feel free to suggest additional FAQs by contacting `support@concordium.software`.
+
+.. dropdown::  Example question?
+
+    Example text
+
+    Example picture
+
+    .. image:: ./images/onboarding_ethereum_developers_1.png
+      :width: 100 %
+
+    Example code
+
+    .. code-block:: console
+
+        $./concordium-client contract init piggy_bank_part2_module --sender <account-name> --contract PiggyBank --name piggy_bank_part2_instance --energy 1000 --grpc-port 10001
+
+    Example note
+
+    .. note::
+
+        abcdefgh
+
+Concordium smart contracts:
+===========================
+
+.. dropdown::  What smart contract language is used on Concordium?
+
+    Concordium uses Rust as smart contract language.
+
+.. dropdown::  How can I start with the Rust smart contract language?
+
+    Rust is a fast and memory-efficient
+    language that is a popular smart contract language among different blockchain projects.
+    There is plenty of literature to get started with Rust such as the
+    `Rust language book <https://doc.rust-lang.org/book/>`_.
+    You can find examples of smart contracts in the
+    `Concordium Rust smart contract repo  <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/examples>`_.
+
+.. dropdown::  Where can I find example smart contracts?
+
+    You can find examples of smart contracts in the
+    `Concordium Rust smart contract repo  <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/examples>`_.
+
+.. dropdown::  Do you have a smart contract reference library similar to the GitHub repo from `OpenZeppelin`?
+
+    You can find examples and standard implementations in the
+    `Concordium Rust smart contract repo  <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/examples>`_.
```

Comment:
Isn't openZeppelin meant to be more vetted than what we have? We explicitly say "don't use these examples for production".

Maybe it would be good to point that out here.